docs/index.rst

```diff
@@ -71,23 +94,13 @@ please go to psycopg docs: http://initd.org/psycopg/docs/
 .. note::
 
    Throughout this documentation, examples utilize the `async/await` syntax
-   introduced by :pep:`492` that is only valid for Python 3.5+.
+   introduced by :pep:`492` that is only valid for Python 3.5.2+.
```

Comment:
I think you can just drop the note.
I made a sense 4 years ago, but now nobody uses `yield from` for asyncio in their code.